app/views/articles/show.html.erb

```diff
@@ -166,8 +166,8 @@
                 <em>
                   Originally published at
                   <a href="<%= @article.canonical_url || @article.feed_source_url %>" style="color:#1395b8"><%= get_host_without_www(@article.canonical_url || @article.feed_source_url) %></a>
-                  on
                   <% if @article.crossposted_at %>
+                    on
```

Comment:
Just moved the "on" into the conditional, since otherwise we'd end up "Originally published at example.com on" without a time stamp.